src/UI/Factory.php

```diff
@@ -336,4 +336,63 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Drop Downs reveal a list of interactions that change the system’s status or navigate to
+	 *      a different view.
+	 *   composition: >
+	 *      Drop Down is a clickable, graphically obtrusive control element. It can
+	 *      bear text.
+	 *   effect: >
+	 *      On-click, a list of actions is revealed. Clicking an item will trigger the action indicated.
+	 *      Clicking outside of an opened drop down will close the list of items.
+	 *   rivals:
+	 *      button: >
+	 *          Buttons are used, if single actions should be presented directly in the user interface.
+	 *      links: >
+	 *          Links are used to trigger Interactions that do not change the systems
+	 *          status. They are usually contained inside a Navigational Collection.
+	 *
+	 *
+	 * rules:
+	 *   usage:
+	 *      1: >
+	 *           Drop Downs MUST NOT be used on their own. They are only parts of more complex UI elements.
+	 *           These elements MUST define their use of drop down. E.g. a List or a Table MAY define that a certain
+	 *           kind of Drop Down is used as part of the UI element.
+	 *   interaction:
+	 *      1: >
+	 *           Only Drop Down Items MUST trigger an action or change a view. The Drop Down trigger element
+	 *           is only used to show and hide the list of Drop Down Items.
+	 *   style:
+	 *      1: >
+	 *           If Text is used inside a Drop Down label, the Drop Down MUST be at least six characters
+	 *           wide.
+	 *   wording:
+	 *      1: >
+	 *           The caption of a Drop Down SHOULD contain no more than two words.
```

Comment:
But the "must not be used on their own" part is also related to the "DropdownItems have their own entry" issue, or not?